controllers/memberoperatorconfig/memberoperatorconfig_controller.go

```diff
@@ -67,6 +84,12 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
-	updateConfig(memberconfig)
+
+	allSecrets, err := loadSecrets(r.Client, request.Namespace)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	updateConfig(memberconfig, allSecrets)
 	return reconcile.Result{}, nil
```

Comment:
you could replace the whole content of the `Reconcile` method with 
```
	return reconcile.Result{}, loadLatest(r.Client, request.Namespace)
```

Reply:
Good point! I did it in f58a325fa6d1f8737001c7621f18db3ca060c54c